RHEL6_7/networking/samba/check

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+
+SOLUTION_FILE="./solution.txt"
+
+echo "SMB1 protocol is deprecated as it is replaced by SMB2 protocol. It is now considered insecure, due to it's vulnerability to man-in-the-middle attacks. Support for the SMB2 protocol will not be backported to Red Hat Enterprise Linux 6 version of samba\* packages. Therefore, for security and compatibility reasons, we strongly recommend to the user to upgrade whole Red Hat Enterprise Linux based server - clients network at once." > $SOLUTION_FILE
+
+log_high_risk "SMB1 protocol is deprecated."
+exit $RESULT_FAIL
```

Comment:
High risk is not needed here as the action is required on the target machine. Use rather the `log_medium_risk` instead.